build-and-test.ps1

```diff
@@ -1,20 +1,26 @@
 #!/usr/bin/env pwsh
 [cmdletbinding()]
 param(
-    [ValidateSet('runtime-sdk', 'aspnet', 'wcf', 'samples')]
-    [string]$ManifestType = 'runtime-sdk',
+    [ValidateSet('default', 'samples')]
```

Comment:
Fixed.  Required some changes to the common script because we need to use multiple path filters to support both runtime and SDK images.

I also removed `samples` because the build for that doesn't work in dev environment due to their being multiple Dockerfiles with the same build context path that are attempted to be built.  This causes an exception in the BuildCommand.  This doesn't happen on official builds because of path filtering to only build for a specific OS.